locations/storefinders/where2getit.py

```diff
@@ -161,3 +164,31 @@ def parse_locations(self, response, **kwargs):
 
     def parse_item(self, item: Feature, location: dict, **kwargs):
         yield item
+
+    def storefinder_exists(response: Response) -> bool | Request:
+        # Example: https://www.att.com/stores/
+        # <script>
+        #     let locationsCount;
+        #     var W2GI = {
+        #         config: {
+        #             appkey: '...',
+        if response.xpath('//script[contains(text(), "W2GI")]').get():
```

Comment:
Per https://www.mattressfirm.com/en-us/stores/ I'd rather this check be a bit more specific e.g. `var W2GI = {` to avoid capturing "W2GI" in some unrelated blob of obfuscated data/code in a 5MB web page response.

Reply:
```suggestion
        if response.xpath('//script[contains(text(), "var W2GI = {")]').get():
```